modules/microshift-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-snapshot_{context}"]
+== Uninstalling the CSI driver and CSI snapshot implementation
+
+To uninstall the installed CSI driver and CSI snapshot implementation, use the following procedure.
+
+.Procedure
+
+. To uninstall the CSI driver implementation, run all the following commands:
```

Comment:
```suggestion

```

Reply:
there were 2 parts :
a. uninstall the CSI driver implementation
b. uninstall the CSI snapshot implementation.

a. has 3 substeps. I have to give one command per code block so had to follow substeps.

Reply:
resolving as per discussion